File: webapp/sandbox/template.py
#surname

import sys
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns

if __name__ == "__main__":

    flu = pd.read_csv("./analysis_data/weekly_data.csv")
    ili = pd.read_csv("./analysis_data/influenza_like_illness.csv")

    flu_and_ili = flu.merge(ili, on = ["MMWR_YR","MMWR_WK"] )
 

File: webapp/sandbox/template.R
#surname

ili_data <- read.csv("analysis_data/influenza_like_illness.csv")
flu_data <- read.csv("analysis_data/weekly_data.csv")

combined_data = merge(ili_data, flu_data, by = c("MMWR_YR", "MMWR_WK", "season", "season_week"))

